packages/airnode-node/src/reporting/heartbeat.ts

```diff
@@ -56,16 +42,18 @@ export async function reportHeartbeat(state: CoordinatorState): Promise<PendingL
   const httpGatewayUrl = getHttpGatewayUrl(config);
   const httpSignedDataGatewayUrl = getHttpSignedDataGatewayUrl(config);
 
-  const heartbeatPayload = {
+  const timestamp = Date.now();
+
+  const heartbeatPayload = JSON.stringify({
```

Comment:
I think it's worth adding a comment explaining why this is stringified. i.e. to avoid any minor differences that might occur when encoding/decoding. The payload/message needs to be exactly the same everywhere

Reply:
I've added the following comment:
```
  /*
  The heartbeat payload is serialised as JSON and then serialised again in JSON as the value of the payload field.
  The reason this is done is to avoid any inconsistencies between different JSON serialisation implementations.
   */
```